cmd/pbstatuser/main.go

```diff
@@ -162,9 +167,11 @@ func checkSourceAvailabilityGCP(ctx context.Context) {
 			sr.Err = err
 			logger.Warn().Msgf("Could not list gcp regions %s", err)
 			chSend <- sr
+			metrics.IncTotalAvailablilityCheckReqs("gcp", string(sr.Status), "statuser", err.Error())
 		} else {
 			sr.Status = kafka.StatusAvaliable
 			chSend <- sr
+			metrics.IncTotalAvailablilityCheckReqs("gcp", string(sr.Status), "statuser", "")
```

Comment:
These conversions to string are ugly, keep the types as long as possible and do the conversion in the function itself. What I mean is use the proper type for the second argument of the function that will be definitely cleaner.